### PR TITLE
add jaykeeb sebelas

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

add new board "jaykeeb sebelas"

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
1. `matrix_diagram.md` and matrix information in info.json layouts don't match:  
`matrix_diagram.md` has one additional key than key count of each community layout in `info.json` layouts.
2. CI builds are failing for community layouts as the layout macro's, for each community layout, has one less key than the key count of the respective community layout.

If the matrix diagram is correct then each layout in `info.json` will need to be amended,
else remove `community layouts` from `info.json`

That'll correct the CI builds 🙂

**Comment by alabahuy** [comment]
@dunk2k thanks, i already remove community layout

**Comment by dunk2k** [comment]
> @dunk2k thanks, i already remove community layout

Great stuff! I've noticed are few other 65% sized board PRs coming in that have the upper right most key removed. I'm starting to wonder if this should be accommodated for 🤔

**Comment by dunk2k** [comment]
@alabahuy 

just noticed you both the `community layouts` field and the layout macros that were named same as community layouts.

in `info.json` could you please revert deletion of the community named layout macros, but not revert the removal of `community layouts` field?

These community named layouts will still function without the `community layout` field in `info.json` and these layouts would be required if the PCB can be populated with more than one switch arrangement, as this PCB can 😊

**Comment by dunk2k** [comment]
@alabahuy 
would you be able to add keymaps for layout macros `LAYOUT_65_ansi_blocker_split_bs`, `LAYOUT_65_ansi_blocker_tsangan`, and `LAYOUT_65_ansi_blocker_tsangan_split_bs`?

`default` keymap is set to layout macro `LAYOUT_65_ansi_blocker`
Suggestions for keymap names:
| layout macro | keymap name |
| --- | --- |
| LAYOUT_65_ansi_blocker_split_bs | blckr |
| LAYOUT_65_ansi_blocker_tsangan | blckr_tsgn  |
|  LAYOUT_65_ansi_blocker_tsangan_split_bs | blckr_tsgn_sb |

**Comment by alabahuy** [comment]
@dunk2k thank you very much, I followed your suggestion and still named the layout macro

File: keyboards/jaykeeb/sebelas/info.json [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+        "alternating": true,
+        "breathing": true,
+        "christmas": true,
+        "knight": true,
+        "rainbow_mood": true,
+        "rainbow_swirl": true,
+        "rgb_test": true,
+        "snake": true,
+        "static_gradient": true,
+        "twinkle": true
```
Comment (zvecr) [inline keyboards/jaykeeb/sebelas/info.json:44]:
```suggestion
        "animations": {
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

File: keyboards/jaykeeb/sebelas/info.json [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+                {"label": ".", "matrix": [3, 10], "x": 10.25, "y": 3},
+                {"label": "/", "matrix": [3, 11], "x": 11.25, "y": 3},
+                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
+                {"label": "\u2191", "matrix": [3, 13], "x": 14, "y": 3},
+                {"label": "Page Down", "matrix": [3, 14], "x": 15, "y": 3},
+
+                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"label": "GUI", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "Space", "matrix": [4, 7], "x": 6, "y": 4, "w": 1.25},
```
Comment (zvecr) [inline keyboards/jaykeeb/sebelas/info.json:123]:
```
linting jaykeeb/sebelas
Ψ Lint check passed!
Keyboard Name: Sebelas
Manufacturer: Alabahuy
Website: 
Maintainer: Alabahuy
Layouts: LAYOUT_65_ansi_blocker, LAYOUT_65_ansi_blocker_split_bs, LAYOUT_65_ansi_blocker_tsangan, LAYOUT_65_ansi_blocker_tsangan_split_bs, LAYOUT_all
Processor: RP20[40](https://github.com/qmk/qmk_firmware/actions/runs/5343882060/jobs/9687556069#step:8:41)
Bootloader: rp2040
Layout Layout_All:
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│Es││1 ││2 ││3 ││4 ││5 ││6 ││7 ││8 ││9 ││0 ││- ││= ││Ba││De│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘
┌────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌────┐┌──┐
│Tab ││Q ││W ││E ││R ││T ││Y ││U ││I ││O ││P ││[ ││] ││\   ││En│
└────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└────┘└──┘
┌─────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌───────┐┌──┐
│Caps ││A ││S ││D ││F ││G ││H ││J ││K ││L ││; ││' ││Enter  ││Pa│
└─────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└───────┘└──┘
┌───┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌─────┐┌──┐┌──┐
│Shi││\ ││Z ││X ││C ││V ││B ││N ││M ││, ││. ││/ ││Shift││↑ ││Pa│
└───┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└─────┘└──┘└──┘
┌───┐┌───┐┌───┐         ┌───┐           ┌───┐┌───┐  ┌──┐┌──┐┌──┐
│Ctr││GUI││Alt│         │Spa│           │Alt││Ctr│  │← ││↓ ││→ │
└───┘└───┘└───┘         └───┘           └───┘└───┘  └──┘└──┘└──┘
```

```suggestion
                {"label": "Space", "matrix": [4, 7], "x": 3.75, "y": 4, "w": 6.25},
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/jaykeeb/sebelas/info.json (hunk 2 above)
Comment (alabahuy) [reply keyboards/jaykeeb/sebelas/info.json]: sorry my fault, i fixed this. thanks

**Review by drashna — APPROVED** [review]

**Comment by alabahuy** [comment]
Any update @zvecr ?

**Review by waffle87 — APPROVED** [review]

**Pull request merged**